FitnesseRoot/HttpTestSuite/FileServerTestSuite/SimpleOption/content.txt

```diff
@@ -1,22 +1,17 @@
-!-<i>OPTIONS to /method_options should return the following
-basic methods: GET,HEAD,POST,OPTIONS,PUT.</i>-!
+!-<i>OPTIONS to any path except /logs should return the
+basic methods: GET,HEAD,POST,OPTIONS,PUT,DELETE.</i>-!
 
 |script              |http browser                                              |
 |set host            |localhost                                                 |
 |set port            |5000                                                      |
-|options             |/method_options                                           |
+|options             |/file1                                                    |
 |ensure              |response code equals           |200                       |
-|ensure              |response header allow contains |GET,HEAD,POST,OPTIONS,PUT |
-|get                 |/method_options                                           |
-|put                 |/method_options                                           |
-|post                |/method_options                                           |
-|head                |/method_options                                           |
-|options             |/method_options2                                          |
+|ensure              |response header allow contains |GET,HEAD,POST,OPTIONS,PUT,DELETE|
```

Comment:
I'm glad this test is getting attention because it's one of the worst in cob_spec. But, I don't think this is the right fix.

The `OPTIONS` method should return the methods that a resource can respond to. If `OPTIONS` tells me that `DELETE` is allowed, but when I try to delete I get a 405, that's wrong. It's not horribly wrong because `OPTIONS` and `Allow` are non-authoritative information. Simply listing all methods shouldn't cause errors, but it's contrary to how this concept is supposed to work and it's not teaching the concept.

Reply:
This was supposed to be inline with the making it more consistently file server-like - I figured that all these methods should be available for files in the public dir (so `DELETE` wouldn't be a `405`, it would be a `200` to signify successfully deleted), but actually as the file already exists the `POST` method shouldn't be an option.